src/select/select-component.js

```diff
@@ -859,20 +859,11 @@ class Select extends React.Component<PropsT, SelectStateT> {
       overrides.SearchIconContainer,
       StyledSearchIconContainer,
     );
-    const [SearchIcon, searchIconProps] = getOverrides(
-      overrides.SearchIcon,
-      SearchIconComponent,
-    );
     const sharedProps = this.getSharedProps();
 
     return (
-      // TODO(v11): remove searchIconProps from SearchIconContainer
-      <SearchIconContainer
-        {...sharedProps}
-        {...searchIconProps}
```

Comment:
The `SearchIcon` override should be maintained but this part should be changed. Notice how this element is getting two different sets of overrides applied to it. View this PR that introduced it for more context https://github.com/uber/baseweb/pull/2397

Sort of tldr on it is one override was previously split into two and to maintain backwards compat the props from the original was applied to both